subsys/bluetooth/host/mesh/cfg_srv.c

```diff
@@ -2200,6 +2200,13 @@ static void node_identity_set(struct bt_mesh_model *model,
 
 		if (IS_ENABLED(CONFIG_BT_MESH_GATT_PROXY)) {
 			sub->node_id = node_id;
+
+			if (node_id) {
```

Comment:
Not a fan of ternary operators right? :) I actually like them for cases like this one

Reply:
I have nothing against them when it improves the readability of the code, i.e. less "brain cycles" needed to quickly grasp it. However, I've mostly found them to be useful as part of function call arguments, when it'd be potentially unclear what the difference is if almost similar calls were made for the two branches of an if-else statement. In this case, at least for myself the code is more readable with if-else. The only thing I might improve is to make an explicit comparison with BT_MESH_NODE_IDENTITY_RUNNING instead of using the value directly as a boolean.

Reply:
Ok, so I tried changing this to ternary, and it was actually quite good looking (at least considering the number of saved lines). So I agree with you here :) The PR has been updated accordingly.